littleblog/entry/views.py

```diff
@@ -1,13 +1,16 @@
 from rest_framework import generics
 from rest_framework import viewsets
-from .serializers import EntryCreateSerializer, UserProfileCreateSerializer, RateUpdateSerializer, EntryDetailSerializer
-from .models import Entry
-from rest_framework.permissions import IsAuthenticated, IsAdminUser, AllowAny
-from .permissons import IsOwnerOrReadOnly, IsAuthenticatedOrWriteOnly
-from rest_framework.response import Response
 from django.contrib.auth import get_user_model
+from rest_framework.decorators import action
+from rest_framework.response import Response
 from rest_framework import status
 
+from .serializers import EntryCreateSerializer, UserProfileCreateSerializer, RatingUpdateSerializer, EntryDetailSerializer
+from .models import Entry
+from rest_framework.permissions import IsAuthenticated
```

Comment:
wong import groupping

Reply:
I forgot that IsAuthenticated is not custom permission. Already moved to the top group